javascript/operators/import.json

```
@@ -72,9 +72,17 @@
               "ie": {
                 "version_added": false
               },
-              "nodejs": {
-                "version_added": false
-              },
+              "nodejs": [
+                {
+                  "version_added": "13.2.0",
+                  "notes": "Dynamic `import` can be used in either CommonJS or ES module files, to import either CommonJS or ES module files. See Node's [ECMAScript Modules documentation](https://nodejs.org/api/esm.html#esm_import_expressions) for more details."
+                },
+                {
+                  "version_added": "12.17.0",
+                  "version_removed": "13.0.0",
+                  "notes": "Dynamic `import` can be used in either CommonJS or ES module files, to import either CommonJS or ES module files. See Node's [ECMAScript Modules documentation](https://nodejs.org/api/esm.html#esm_import_expressions) for more details."
+                }
+              ],
```

Let's avoid duplicating these notes, as parent notes generally also apply to its children.

```suggestion
              "nodejs": [
                {
                  "version_added": "13.2.0"
                },
                {
                  "version_added": "12.17.0",
                  "version_removed": "13.0.0"
                }
              ],
```